e2e-tests/README.md

```diff
@@ -25,11 +25,18 @@ rest of Socorro so you're not mixing requirements:
 	$ # your dev virtualenv.
 	$ pip install -r requirements.txt
 
-An additional constraint for Firefox users, with the release of Firefox 48, Webdriver support is currently broken until GeckoDriver is out of Alpha. We suggest using an older version of Firefox which can be [downloaded here][firefoxdownloads].
+An additional constraint for Firefox users, with the release of Firefox 48, Webdriver support is currently broken until GeckoDriver is out of Alpha. We suggest using an older version of Firefox which can be [downloaded here][firefoxdownloads]. Note that on Linux, you don't need to downgrade your Firefox install in order to use an older version.
```

Comment:
Selenium is compatible with most versions of Firefox. With non-legacy versions, Firefox 48 and above, the new goodness is in using geckodriver. For older versions we can continue to use the selenium client without geckodriver

In our instance, we likely only care to test crash-stats using the latest version of Firefox.